src/Sylius/Bundle/AdminBundle/TwigComponent/Shared/Navbar/UserDropdown.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\TwigComponent\Shared\Navbar;
+
+use Sylius\Component\User\Model\UserInterface;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+final class UserDropdown
```

Comment:
Just an idea,

```suggestion
final class UserMenu
final class UserMenuDropdown
final class NavbarSecurity
```

Main takeaway from this, that we should establish "public" naming scheme for components, e.g. content based, element based, etc